uswds/uswds-site
# USWDS-Site: Add VS Code directories to .gitignore

## Related issue
Closes https://github.com/uswds/uswds-site/issues/1724

## Problem statement
The `.vscode` directory is currently being tracked by Git. We should include VS Code directories in the project's `.gitignore` file. 

## Testing and review
If you have the peacock extension installed in VS Code,
1. Open VS Code and enter `command` + `shift` + `p`
2. In the command input, run "Peacock: Change color to a favorite color"
3. Select any color
4. A new file will be created --> `.vscode/settings.json`. If working correctly, this file will not appear in the tracked changes.

---

Before opening this PR, make sure you’ve done whichever of these applies to you:
